services/iota_wrappers.go

```diff
@@ -455,10 +471,10 @@ func verifyChunkMessagesMatchRecord(chunks []oyster_utils.ChunkData) (filteredCh
 func verifyChunksMatchRecord(chunks []oyster_utils.ChunkData, checkTrunkAndBranch bool) (filteredChunks FilteredChunk, err error) {
 
 	filteredChunks = FilteredChunk{}
-	addresses := make([]giota.Address, 0, len(chunks))
+	addresses := make([]trinary.Hash, 0, len(chunks))
```

Comment:
here

Reply:
It is not clear to me about this comment